docs/commerce/2.x/en/managing-a-catalog/creating-and-managing-products/channels/configuring-product-visibility-by-site-using-channels.md

```diff
@@ -1,6 +1,6 @@
-# Configuring Product Visibility by Site Using Channels
```

Comment:
Is there a reason why this is in the channels section of the docs and not products section + combined with the account-group filtering? It seems to me to more naturally fit in products, not channels.